client/reader/post-editor/index.jsx

```diff
@@ -0,0 +1,78 @@
+import IsoloatedEditor, { ToolbarSlot } from '@automattic/isolated-block-editor';
+import { serialize } from '@wordpress/blocks';
+import { Button } from '@wordpress/components';
+import { select, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
```

Comment:
Is there a good reason to use `@wordpress/i18n` and not `i18n-calypso`?

Reply:
Nope, just habit. Now changed to use the Calypso package.